src/modules/launcher/Wox.Plugin/SharedCommands/FilesFolders.cs

```diff
@@ -51,9 +52,10 @@ public static void Copy(this string sourcePath, string targetPath)
 #pragma warning restore CS0168 // Variable is declared but never used
             {
 #if DEBUG
-                throw e;
+                throw;
 #else
-                System.Windows.MessageBox.Show(string.Format("Copying path {0} has failed, it will now be deleted for consistency", targetPath));
+                // Using CurrentCulture since this is user facing
+                System.Windows.MessageBox.Show(string.Format(CultureInfo.CurrentCulture, "Copying path {0} has failed, it will now be deleted for consistency", targetPath));
```

Comment:
These messages should be localized.

Reply:
good catch

Reply:
I've created a separate issue to address this- https://github.com/microsoft/PowerToys/issues/7499

Reply:
PR has been updated to add localization for hardcoded messages in this file